diff --git a/src/flight/SMACCMPilot/Flight/Param.hs b/src/flight/SMACCMPilot/Flight/Param.hs
index 685e4d2..41eb741 100644
--- a/src/flight/SMACCMPilot/Flight/Param.hs
+++ b/src/flight/SMACCMPilot/Flight/Param.hs
@@ -87,13 +87,13 @@ data FlightParams f = FlightParams
 -- | Initialize flight parameters to their default values.
 flightParams :: Monad m => ParamT f m (FlightParams f)
 flightParams =                          -- P     I     D     IMAX (-IMIN)
-  FlightParams <$> group "RLL" (stabParams 2.500 0.000 0.000 8.0
-                                           0.100 0.000 0.095 0.5)
-               <*> group "PIT" (stabParams 2.500 0.000 0.000 8.0
-                                           0.100 0.000 0.095 0.5)
+  FlightParams <$> group "RLL" (stabParams 2.000 0.000 0.000 8.0
+                                           0.100 0.000 0.000 0.5)
+               <*> group "PIT" (stabParams 2.000 0.000 0.000 8.0
+                                           0.105 0.000 0.000 5.0)
                <*> param "PITRLL_UI_SENS"  45.0
                <*> group "YAW" (stabParams 1.0   0.000 0.000 8.0
-                                           0.300 0.0   0.0   1.0)
+                                           0.305 0.0   0.0   8.0)
                <*> param "YAW_UI_SENS"     180.0
                <*> group "ALT" altitudeParams
                <*> group "POS" posCtlParams
